docs/devenv-local.md

```diff
@@ -20,7 +20,7 @@ Starting from a default Fedora 26 Server install the following
 extra software needs installing and configuring
 
 ```bash
-sudo dnf install etcd docker git libvirt-devel
+sudo dnf install etcd docker git libvirt-devel rsync
```

Comment:
I don't recall anything needing rsync ?  Could you say what/where you get an error without it ?

Reply:
When I started the script to build k8s - It seems to be a prerequisite [0]

[0] https://github.com/kubernetes/kubernetes/blob/master/build/common.sh#L167

Reply:
Ok I checked my test dev env and it seems that I got rsync installed by the Fedora installer when creating the guest. So I guess you must have selected different install options. Lets just add it here to be safe against this difference.

Reply:
Yes probably, for information I provisioned my environment with a Fedora 26 Server edition.